streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -17,116 +17,109 @@
 
 package org.apache.kafka.streams.kstream;
 
+import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.common.serialization.Serde;
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.processor.ProcessorSupplier;
 
 /**
- * KStream is an abstraction of an event stream in key-value pairs.
+ * KStream is an abstraction of a <i>record stream</i> in key-value pairs.
  *
  * @param <K> Type of keys
  * @param <V> Type of values
  */
+@InterfaceStability.Unstable
 public interface KStream<K, V> {
 
     /**
-     * Creates a new instance of KStream consists of all elements of this stream which satisfy a predicate
+     * Create a new instance of {@link KStream} consists of all elements of this stream which satisfy a predicate.
```

Comment:
As you would expect, Kafka has both forms. It does seem to favour the `Create` form more when looking at `KafkaConsumer` and `KafkaProducer`. It's a bit sad because we are unlikely to ever achieve consistency by choosing something different from the Java style guide (contributors will invariably use the Java recommended style and it's hard to spot all of these during code review).